### PR TITLE
Market monitoring script

### PR DESCRIPTION
**Pull request by mhkhan27**
Hi, Zack, I have added scripts for recoding, daily monitoring, and basic analysis. Please pull

### CODE REVIEW

File: BGD_2020_Markets_Covid/scripts/basic_analysis.R [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+
+
+dfsvy$variables$upazilla<- forcats::fct_expand(dfsvy$variables$upazilla,c( "ukhiya", "teknaf"))
+dfsvy$variables$rice_unit<- forcats::fct_expand(dfsvy$variables$rice_unit,c( "yes", "no"))
+dfsvy$variables$lentils_unit<- forcats::fct_expand(dfsvy$variables$lentils_unit,c( "yes", "no"))
+dfsvy$variables$leafy_greens_unit<- forcats::fct_expand(dfsvy$variables$leafy_greens_unit,c( "yes", "no"))
+dfsvy$variables$bananas_unit<- forcats::fct_expand(dfsvy$variables$bananas_unit,c( "yes", "no"))
+dfsvy$variables$soap_unit<- forcats::fct_expand(dfsvy$variables$soap_unit,c( "yes", "no"))
+dfsvy$variables$paracetamol_unit<- forcats::fct_expand(dfsvy$variables$paracetamol_unit,c( "yes", "no"))
+dfsvy$variables$sell_tarpaulin<- forcats::fct_expand(dfsvy$variables$sell_tarpaulin,c( "yes", "no"))
```
Comment (zackarno) [inline BGD_2020_Markets_Covid/scripts/basic_analysis.R]:
```suggestion
cols_to_refactor_yn<- c("rice_unit" , "lentils_unit" , "leafy_greens_unit" ,"bananas_unit"," paracetamol_unit" , "soap_unit","sell_tarpaulin")
dfsvy$variables<-dfsvy$variables %>%
    mutate_at(.vars=cols_to_refactor_yn, .fun=forcats::fct_expand(.,c("yes","no"))



```

File: BGD_2020_Markets_Covid/scripts/functions/GIS_Project_Functions.R [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+}
+towgs84<-function(spatial_object){
+  spatial_object_wgs<-spTransform(spatial_object, CRS("+proj=longlat +datum=WGS84 +no_defs +ellps=WGS84 +towgs84=0,0,0"))
+  return(spatial_object_wgs)
+}
+
+
+spatial_join<-function(point,poly){
+  if (class(point)=="SpatialPoints"){
+    df_over<-over(point,poly)
```
Comment (zackarno) [inline BGD_2020_Markets_Covid/scripts/functions/GIS_Project_Functions.R:13]: you are using functions from the sp package to make new functions.  I recommend forgetting about the sp package and only use sf package

File: BGD_2020_Markets_Covid/scripts/functions/NAResponses.R [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+}
+
+
+na_count <-sapply(HeaderYes, function(y) sum(length(which(is.na(y)))))
+na_perc <-sapply(HeaderYes, function(y) ((sum(length(which(is.na(y)))))/nrow(HeaderYes))*100)
+non_response_df<-data.frame(num_non_response=na_count,perc_non_response= na_perc)
+non_response_df1<-non_response_df %>%
+  mutate(question=rownames(.)) %>% 
+  dplyr::select(question, everything()) %>% 
+  arrange(num_non_response, question)
```
Comment (zackarno) [inline BGD_2020_Markets_Covid/scripts/functions/NAResponses.R:14]: just use butteR::get_na_response_rates

Comment (mhkhan27) [reply BGD_2020_Markets_Covid/scripts/functions/NAResponses.R:14]: Ok I will.

**Pull request merged**